compiler/src/main/kotlin/software/amazon/lastmile/kotlin/inject/anvil/Util.kt

```diff
@@ -24,3 +27,32 @@ internal fun <T : Annotatable.Builder<T>> Annotatable.Builder<T>.addOriginAnnota
         .addMember("value = %T::class", clazz.toClassName())
         .build(),
 )
+
+internal inline fun <reified T> KSAnnotation.argumentOfTypeAt(
+    context: ContextAware,
```

Comment:
When this is a parameter, then I kept these utility functions in `ContextAware` itself.